src/analysis/souffle/taint.dl

```diff
@@ -52,7 +52,8 @@ hasTag(tgt, tag) :- claimHasTag(principal, tgt, tag), ownsTag(principal, tag).
 mayHaveTag(tgt, tag) :- hasTag(tgt, tag).
 
 mayHaveTag(tgt, tag) :-
-  edge(src, tgt), mayHaveTag(src, tag), !downgrades(tgt, tag).
+    edge(src, tgt), mayHaveTag(src, tag),
+    !downgrades(tgt, tag).
```

Comment:
Nit: Why put `!downgrades(tgt, tag)` on its own line? There's plenty of room on the previous line with a line length of 80 characters. Gogul and I agreed to make the line length 100 characters in Datalog to further reduce the number of newlines. I think this was early on though, so it might not have been stated in a group chat.